### PR TITLE
write specification for eda_text_columns function along with dependencies

### LINKED ISSUE #9: eda_text_columns - Write Specification
For this milestone you will write function documentation that will serve as specifications, but you will NOT write any code for your package functions. Specifically, set-up empty functions (with appropriate function names) containing no code in the files where you will eventually write code. Write function documentation (e.g., complete docstrings). These are the functions you wrote about in your proposal.

TODO: replace Function 3 with an actual function name

### PR DESCRIPTION
**Pull request by charlessuresh**
Did the following:
- wrote function documentation / specifications for eda_text_columns function
- set-up empty functions (with appropriate function names) containing no code
- added import statements needed for eda_text_columns function

@adibns is my dedicated reviewer for this week.
@lephanthuymai @rahulkuriyedath please feel free to suggest / review / comment

### CODE REVIEW

**Comment by adibns** [comment]
It is good

File: datascience_eda/datascience_eda.py [hunk 1]
```diff
@@ -90,4 +95,73 @@ def initial_clustering(
 
     # add all plots to result dictionary
 
-    return result
\ No newline at end of file
+    return result
+
+
+def eda_text_columns(df, text_col = None, params = dict()):
+    """Perfroms EDA of text features.
```
Comment (lephanthuymai) [inline datascience_eda/datascience_eda.py:102]: grammar mistake

File: datascience_eda/datascience_eda.py [hunk 2]
```diff
@@ -90,4 +95,73 @@ def initial_clustering(
...
+    df : pandas.DataFrame
+        the dataset (X)
+    text_col : str
+        name of text column
+    params : dict
+        a dictionary of parameters
+
+    Returns
+    -------
+    None
```
Comment (lephanthuymai) [inline datascience_eda/datascience_eda.py:126]: Tiff said we should always return something, probably return a list of plots?

File: datascience_eda/datascience_eda.py (hunk 1 above)
Comment (charlessuresh) [reply datascience_eda/datascience_eda.py:102]: Thanks for pointing it out! Addressed this in the new [PR](https://github.com/UBC-MDS/datascience_eda/pull/17)

File: datascience_eda/datascience_eda.py (hunk 2 above)
Comment (charlessuresh) [reply datascience_eda/datascience_eda.py:126]: Addressed this in the new [PR](https://github.com/UBC-MDS/datascience_eda/pull/17)

**Pull request merged**